site/_wiki/FAQ/General.md

```diff
@@ -82,11 +82,11 @@ Uninstall any other tablet drivers you have installed.
 
 ### How to convert areas to and from OpenTabletDriver? {#area-conversion}
 
-#### Conversion through the UI
+#### Conversion through the OpenTabletDriver UI
 
-- Right click the tablet area editor
-- Click the `convert item` menu item
-- Select the OEM driver and insert your area values
+- Right click the Tablet Area Editor
+- Click the `Convert Item` menu item
+- Select your target OEM driver, and insert the appropriate values
```

Comment:
Conversion goes from OEM driver to OpenTabletDriver, "target" here doesn't feel right. Maybe something like:

```suggestion
- Select the OEM driver to convert from, then input the appropriate values to the converter
```

Reply:
Good catch, I had the usage flipped.